l10n/configs/pontoon.toml

```diff
@@ -247,3 +247,11 @@ locales = [
 [[paths]]
     reference = "en/privacy/*.ftl"
     l10n = "{locale}/privacy/*.ftl"
+[[paths]]
+    # TODO - Limit VPN pages to locales where VPN has launched
+    reference = "en/products/vpn/**/*.ftl"
+    l10n = "{locale}/products/vpn/**/*.ftl"
+    locales = [
```

Comment:
I know we want to restrict the languages from being exposed in Pontoon. Do I make that determination here or in the [`/l10n/pontoon.toml`](https://github.com/mozilla/bedrock/blob/9964-stub-vpn-seo-pages/l10n/configs/pontoon.toml) file?

Reply:
This will expose the strings in Pontoon for German and French, which is not something we want to do just yet. Really until we're ready to localize these pages, we should keep the strings out of Pontoon (this is why the current landing page is hard-coded). 

I think it might be a good idea for us to do a sycn to make sure we're all on the same page for what is happening for the VPN launch.

/cc @rraue

Reply:
I removed this per our conversation earlier with @peiying2.